### ISSUE TITLE
HTML save() with bokeh discards colors, labels

### ISSUE THREAD

**Issue by hobzcalvin** [ioam/holoviews#3067]
Consider the bokeh Chord example notebook:
https://github.com/ioam/holoviews/blob/master/examples/reference/elements/bokeh/Chord.ipynb

But replace the last line:
`hv.Chord((links, nodes)).select(value=(5, None))`
with a `save()` like so:
```
obj = hv.Chord((links, nodes)).select(value=(5, None))
hv.renderer('bokeh').save(obj,'out') # copied from http://holoviews.org/user_guide/Deploying_Bokeh_Apps.html
```

Colors and labels show up just fine when rendered to the notebook:
<img width="960" alt="screen shot 2018-10-09 at 10 28 30 am" src="https://user-images.githubusercontent.com/82265/46686829-2d539300-cbae-11e8-9281-ef6edbce1bcc.png">
but the output HTML is missing these:
<img width="632" alt="screen shot 2018-10-09 at 10 28 53 am" src="https://user-images.githubusercontent.com/82265/46686837-32184700-cbae-11e8-923a-cc822897a79a.png">

I tried something like this in case this plot needs a running server (though I don't know why it would; does this use DynamicMap??):
```
server = renderer.app(dmap, show=True, new_window=True)
```
Didn't seem to help.

**Comment by hobzcalvin** [comment]
Inlining the server doesn't work either:
<img width="960" alt="screen shot 2018-10-09 at 1 39 04 pm" src="https://user-images.githubusercontent.com/82265/46697152-dc9d6380-cbc8-11e8-862f-821e0550da40.png">
not sure if it's relevant, but I see these errors in the JS console when inlining:
<img width="1068" alt="screen shot 2018-10-09 at 1 39 46 pm" src="https://user-images.githubusercontent.com/82265/46697175-e8892580-cbc8-11e8-92ab-99526f2f0402.png">

**Comment by hobzcalvin** [comment]
Are you saying the .options syntax will persist these features properly?

**Comment by philippjfr** [comment]
Yes, that's right, alternatively you can also display the object normally in one cell and then save it in another. The magics work by customizing the objects as they are being displayed in the notebook so without displaying them first the customization is not applied.

So your options are:

```python
options = dict(label_index='name' color_index='index' edge_color_index='source',
    cmap='Category20' edge_cmap='Category20')
chord = hv.Chord((links, nodes)).select(value=(5, None)).options(**options)
hv.renderer('bokeh').save(chord, 'chord.html')
```

Or display it in one cell:

```python
%%opts Chord [label_index='name' color_index='index' edge_color_index='source'] 
%%opts Chord (cmap='Category20' edge_cmap='Category20')
chord = hv.Chord((links, nodes)).select(value=(5, None))
chord
```

and save in the next:

```python
hv.renderer('bokeh').save(chord, 'chord.html')
```

**Comment by hobzcalvin** [comment]
That works, thanks @philippjfr!
It looks like the Chord docs page http://holoviews.org/reference/elements/bokeh/Chord.html is auto-generated from a notebook file https://github.com/ioam/holoviews/blob/master/examples/reference/elements/bokeh/Chord.ipynb so how does the Holoviews web site render the interactive charts properly?

I ask because I need to embed several charts amongst other HTML so I'm concerned about namespace conflicts etc. if I try to just paste multiple saved HTML blobs into a single document. It looks like the docs are just rendering a notebook in full, which could work too I suppose...unfortunately this is all inside Wordpress...

**Comment by philippjfr** [comment]
It is true that most of our docs are generated directly from the notebook, which is done using a sphinx extension. We also have a project explicitly to build a website from notebooks called nbsite but if you're simply embedding plots that's probably not that useful to you.

>I ask because I need to embed several charts amongst other HTML so I'm concerned about namespace conflicts etc. if I try to just paste multiple saved HTML blobs into a single document.

I've done this in the past without issues. I also had an approach that exported pure JS files which can then be loaded in a script tag, which I made use of here: http://blog.holoviews.org/release_1.5.html which was generated from https://github.com/pyviz/blog/blob/master/content/gv_release_1.5.ipynb
